### ISSUE TITLE
SOLID Services - the Python way 

### ISSUE THREAD

**Issue by AnasKiron** [dotnetjunkie/solidservices#36]
Hi Steven,
I search the web for the Python way of doing SOLID services. 

I found no hits. Do you have any information in this regards, please?

**Comment by dotnetjunkie** [comment]
Nope. I've got exactly 0 experience with Python. You might be the first Pythoneer trying this. Good luck in your quest and don't forget to blog about your experiences or create a Github repo with demo code.
